### ISSUE TITLE
When it will support scan card?

### ISSUE THREAD

**Issue by gaofeiyang2022** [stripe/stripe-react-native#1259]
When it will support scan cards?

I saw stripe-ios and android has supported scan

**Comment by charliecruzan-stripe** [comment]
Payment Sheet supports card scanning, have you tried using Payment Sheet?

**Comment by crechet** [comment]
Hi! Our team wants to add the credit card scanning feature. Currently we are using the CardField component to collect user’s card data. Is there a way to use card scanning both for iOS and Android with CardField component?

**Comment by charliecruzan-stripe** [comment]
Not with the CardField component, yet. You'd need to use the Payment Sheet

**Comment by crechet** [comment]
> Not with the CardField component, yet. You'd need to use the Payment Sheet

It's sad. Thanks!

**Comment by gaofeiyang2022** [comment]
Yes, I also hope this scanning feature can be integrated into CardField Component.

**Comment by robertcreaner** [comment]
Agreed, would be awesome to have the Card Scan feature available on the CardForm/CardField components

**Comment by theodoregold** [comment]
Is there any roadmap when this could be available?

**Comment by trilam1409** [comment]
My app uses Stripe to add cards on the backend side. On the client side, we add cards on our custom form. So we need to add a feature to support getting card information via scan. Hope the feature will be released soon.

**Comment by tuanlevi95** [comment]
i saw its support on payment sheet, but only ios, android not support

**Comment by sugerGDev** [comment]
Flutter 应用，已集成stripe 支付并且自定义UI，现在需要单独使用信用卡扫描功能在Android和IOS设备上，为何不行？ 还必须要用PaymentSheet？
这样完全破坏自定义UI 的功能！

**Comment by AlixH** [comment]
Up

**Comment by volkanakbayir** [comment]
I also observed that, in payment sheet scan card button is only available in ios and is not visible on android

**Comment by ElCarsti** [comment]
+1

**Comment by Shoaib-Raza** [comment]
@charliecruzan-stripe Any update on scanning option for CardField component? As we don't use the Payment Sheet option because we only need the option for managing cards. 

A response would be really appreciated. Thanks

**Comment by Under-Warz** [comment]
+1

**Comment by wooj-stripe** [comment]
@gaofeiyang2022 . Sorry for the delay. At this time, we are not planning on adding any new features to the Card Element. We are encouraging all users to instead use the Mobile Payment Element, which we launched in 2021. You can find the docs [here](https://docs.stripe.com/payments/accept-a-payment). If there are any reasons the MPE doesn't support your use case, we'd love to hear your feedback!
